groovy/inductor_dashboard.groovy

```diff
@@ -0,0 +1,376 @@
+                [$class: 'StringParameterValue', name: 'tag', value: "${image_tag}"],
+            ]
+            task_status = image_build_job.result
+            task_number = image_build_job.number
+            withEnv(["task_status=${task_status}","task_number=${task_number}"]) {
+                sh '''#!/bin/bash
+                    tag=${image_tag}
+                    old_container=`docker ps |grep pt_inductor |awk '{print $1}'`
+                    if [ -n "${old_container}" ]; then
+                        docker stop $old_container
+                        docker rm $old_container
+                        docker container prune -f
+                    fi
+                    old_image_id=`docker images|grep pt_inductor|grep ${tag}|awk '{print $3}'`
+                    old_image=`echo $old_image_id| awk '{print $1}'`
+                    if [ -n "${old_image}" ]; then
+                        docker rmi -f $old_image
+                    fi
+                    if [ ${task_status} == "SUCCESS" ]; then
+                        docker system prune -f
+                        docker login ccr-registry.caas.intel.com
+                        docker pull ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag}
+                    fi
+                '''
+            }           
+        }else {
+            sh '''
+            #!/usr/bin/env bash
+            tag=${image_tag}
+            old_container=`docker ps |grep pt_inductor |awk '{print $1}'`
+            if [ -n "${old_container}" ]; then
+                docker stop $old_container
+                docker rm $old_container
+                docker container prune -f
+            fi
+            old_image_id=`docker images|grep pt_inductor|grep ${tag}|awk '{print $3}'`
+            old_image=`echo $old_image_id| awk '{print $1}'`
+            if [ -n "${old_image}" ]; then
+                docker rmi -f $old_image
+            fi
+            docker system prune -f
+            mv docker/Dockerfile ./
+            DOCKER_BUILDKIT=1 docker build --no-cache --build-arg http_proxy=${http_proxy} --build-arg https_proxy=${https_proxy} --build-arg BASE_IMAGE=${BASE_IMAGE}--build-arg PT_REPO=${PT_REPO} --build-arg PT_BRANCH=${PT_BRANCH} --build-arg PT_COMMIT=${PT_COMMIT} --build-arg TORCH_VISION_BRANCH=${TORCH_VISION_BRANCH} --build-arg TORCH_VISION_COMMIT=${TORCH_VISION_COMMIT} --build-arg TORCH_DATA_BRANCH=${TORCH_DATA_BRANCH} --build-arg TORCH_DATA_COMMIT=${TORCH_DATA_COMMIT} --build-arg TORCH_TEXT_BRANCH=${TORCH_TEXT_BRANCH} --build-arg TORCH_TEXT_COMMIT=${TORCH_TEXT_COMMIT} --build-arg TORCH_AUDIO_BRANCH=${TORCH_AUDIO_BRANCH} --build-arg TORCH_AUDIO_COMMIT=${TORCH_AUDIO_COMMIT} --build-arg TORCH_BENCH_BRANCH=${TORCH_BENCH_BRANCH} --build-arg TORCH_BENCH_COMMIT=${TORCH_BENCH_COMMIT} --build-arg BENCH_COMMIT=${BENCH_COMMIT} -t ccr-registry.caas.intel.com/pytorch/pt_inductor:${tag} -f Dockerfile --target image .

```

Comment:
Why we still need build docker image in test node? I think if we choose don't build image, we should try to pull the image directly, if the pull failed, then the job failed. How do you think?

Reply:
Agree. Previous considerations.